### DISCUSSION TITLE
outdated info: custom_card_esh_room

### DISCUSSION THREAD

**Discussion by mdb17** [UI-Lovelace-Minimalist/UI#864] [Q&A] [answered]
I am new to this project but would like to help in anyway I can. 

I noticed that: https://ui-lovelace-minimalist.github.io/UI/usage/custom_cards/custom_card_esh_room/ is outdated compared to the latest update. I would be willing to update it to represent the newest changes if somebody can walk me through how to get started.

**Comment by basbruss** [comment, marked as answer]
The wiki is updated. There was some error in one of workflows that prevented to publish the newest versions of documentation.
